doc/content/index.md

```diff
@@ -15,6 +15,7 @@ Users will need to request access to the controlled NEAMS software from the [Nuc
 
 - [How to use the Virtual Test Bed?](resources/how_to_use_vtb.md) \\
 - [How to run a Virtual Test Bed example with the NEAMS Workbench on Sawtooth?](neams-workbench.md) \\
+- [How to run tests for the Virtual Test Bed](resources/running_vtb_tests.md) \\
```

Comment:
```suggestion
```
that s too technical imo for the front page

Reply:
What about maybe generalizing "Contribution guidelines to add a new model" to "How to contribute to the Virtual Test Bed" and then on that page, have my stuff as a new section?

Reply:
seems good to me
new page can have then:
- guidelines
- template
- testing